week1/study_materials/4_optional_parameters.md

```diff
@@ -0,0 +1,80 @@
+### Difference between optional and standard parameters.
+
+First of all we must know the difference between parameters and arguments:
+
+A parameter is a variable in a method definition. When a method is called, the arguments are the data you pass into the method's parameters.
+Parameter is variable in the declaration of function. Argument is the actual value of this variable that gets passed to function.
```

Comment:
Parameter is variable `a`? 
`The` argument is the actual..? 
passed to `the` function?